src/util/settings/connection_client.cc

```diff
@@ -213,14 +214,16 @@ QString serverScriptDefault() {
   server_script = qApp->applicationDirPath() + "/../veles-server/srv.py";
 #elif defined(Q_OS_LINUX)
   server_script = qApp->applicationDirPath() + "/../share/veles-server/srv.py";
-#else
+#elif defined(Q_OS_MAC)
   server_script = qApp->applicationDirPath() + "/../Resources/veles-server/srv.py";
+#else
+  #error OS not supported
```

Comment:
Sorry I wasn't specific enough - I meant situation in this ifdef sequence https://github.com/codilime/veles/blob/master/src/ui/connectionmanager.cc#L175.